client/components/forms/form-country-select/index.jsx

```diff
@@ -34,7 +34,7 @@ export default localize( React.createClass( {
 
 		return (
 			<select
-				{ ...omit( this.props, 'className' ) }
+				{ ...omit( this.props, [ 'className', 'countriesList', 'translate', 'moment', 'numberFormat' ] ) }
```

Comment:
Ok, Nope, changing my mind again :)

My whitelist broke the tests for `FormPhoneInput`, because it's useing the deprecated `valueLink` property [here](https://github.com/Automattic/wp-calypso/blob/1480e3cf8280faf52e4365517cac42b260b9299e/client/components/forms/form-phone-input/index.jsx#L76), but thankfully has a test that checks that changing the select value works.

That component also uses `disabled` and passes down values from it's parents, including `Security2faSMSSettings` which is passing down an `onFocus` used for analytics ([here](https://github.com/Automattic/wp-calypso/blob/1480e3cf8280faf52e4365517cac42b260b9299e/client/me/security-2fa-sms-settings/index.jsx#L191)).

That's probably everything, but I nearly broke it all, so I'm going to go back to the filtering approach